frontend/src/theme/utils.ts

```diff
@@ -310,17 +337,14 @@ export const createEmotionTheme = (
 
   const parsedFont = fontEnumToString(font)
 
-  const parsedColors = Object.entries(customColors).reduce(
-    (colors: Record<string, string>, [key, color]) => {
-      if (isColor(color)) {
-        colors[key] = color
-      } else if (isColor(`#${color}`)) {
-        colors[key] = `#${color}`
-      }
-      return colors
-    },
-    {}
-  )
+  const parsedColors: Record<string, string> = {}
```

Comment:
I would vote to keep the old `reduce` here.

Reply:
Will do. This one's a bit weird since I generally prefer using `reduce`, but semantically you'd expect to see the reducer function return a new accumulator object instead of mutating the one given to it, and doing that here seemed like overkill given how much more verbose it ends up being.

But yeah, it's a minor enough thing that I'm happy leaving this unchanged for now / can file another PR to have this follow more normal reduce semantics